Lib/enum.py

```diff
@@ -371,7 +334,7 @@ class _EnumDict(dict):
     """
     def __init__(self):
         super().__init__()
-        self._member_names = {} # use a dict to keep insertion order
+        self._member_names = {} # use a dict -- faster look-up than a list, and keeps insertion order since 3.7 (?)
```

Comment:
```suggestion
        self._member_names = {} # use a dict -- faster look-up than a list, and keeps insertion order
```
Yes, since 3.6 and 3.7 (implemented and documented)

Reply:
Cool, question mark removed.  IIRC it wasn't a language guarantee until 3.7.